test/run

```diff
@@ -63,6 +86,9 @@ def add_e2e_tests(litani, test_dir, root_dir, counter):
         if test_file.name in ["__init__.py", "__pycache__"]:
             continue
 
+        if fast and get_test_module(test_file).SLOW:
```

Comment:
Please check `SLOW` has been defined in the module. It is an error if the variable does not exist.